.gitignore

```diff
@@ -6,6 +6,9 @@ __pycache__/
 # IDE files
 .idea/
 
+# Config files
+money_bot/local_config.py
```

Comment:
This config files was added before, check the [.gitignore](https://github.com/summer-school-2019/money-bot/blob/670eab2f770b37056405a2c4ff5a3b47d52c5951/.gitignore#L109-L110) before your PR

https://github.com/summer-school-2019/money-bot/blob/670eab2f770b37056405a2c4ff5a3b47d52c5951/.gitignore#L109-L110

Leave only one occurrence pls

Reply:
I was very sleepy)